CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v1.2.1
+
+ENHANCEMENTS:
+
+- `resource/auth0_role`: Update role permission and permissions documentation ([#963](https://github.com/auth0/terraform-provider-auth0/pull/963))
```

Comment:
This isn't really an enhancement in terms of functionality. Perhaps we can add it under a NOTES section and be more specific in terms of what was changed / added.